### PR TITLE
add files and notes

### PR DESCRIPTION
**Pull request by janecyyu**
add jsjsjsjs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
 # Test
+//test
```
Comment (anson0910) [inline README.md]: This should be upper case

File: newfile.js [hunk 2]
```diff
@@ -0,0 +1 @@
+//jsjsjs
```
Comment (anson0910) [inline newfile.js]: Upper case as well.

File: README.md (hunk 1 above)
Comment (janecyyu) [reply README.md]: Done

File: newfile.js (hunk 2 above)
Comment (janecyyu) [reply newfile.js]: won't fix

**Pull request merged**